metis/webpack.config.js

```diff
@@ -88,5 +87,14 @@ module.exports = {
       filename: 'public/css/metis.bundle.css',
       allChunks: true,
     }),
+    new CopyPlugin([
```

Comment:
ServiceWorkers must be sideloaded as seperate files, so we are forced to generate the asset outside the bundle.